pyproject.toml

```diff
@@ -20,3 +20,9 @@ exclude = '''
 )/
 '''
 
+[tool.pytest.ini_options]
```

Comment:
According to this, pytest.ini will be preferred over all other config files:
https://docs.pytest.org/en/latest/reference/customize.html#pytest-ini

And we do have [pytest.ini](https://github.com/dbcli/pgspecial/blob/master/tests/pytest.ini). If you want to move the settings, you'd have to delete that.

Reply:
I believe I did: https://github.com/dbcli/pgspecial/pull/111/files?file-filters%5B%5D=.ini :innocent:

Reply:
Hah. :) Great. I must have missed that.